ScoutSuite/providers/aws/resources/iam/users.py

```diff
@@ -0,0 +1,20 @@
+from ScoutSuite.providers.aws.resources.resources import AWSResources
+
+
+class Users(AWSResources):
+    async def fetch_all(self, **kwargs):
+        raw_users = await self.facade.iam.get_users()
+        for raw_user in raw_users:
+            name, resource = self._parse_user(raw_user)
+              
+            if name in self:
```

Comment:
Can we really end up with duplicates in `raw_users` ?!

Reply:
I'm not sure... it was like that before, so I left it. :/ Do you think we should remove this?

Reply:
Nevermind, let's not remove it to avoid unexpected side effects...